tests/ha/ha_cluster_init.pm

```diff
@@ -53,7 +55,12 @@ sub run {
     exec_csync;
 
     # State of SBD if shared storage SBD is used
-    assert_script_run "sbd -d $sbd_device list" unless (get_var('USE_DISKLESS_SBD'));
+    if (!get_var('USE_DISKLESS_SBD')) {
+        $num_nodes  = get_node_number;
+        $sbd_output = script_output("sbd -d $sbd_device list");
+        record_info('bsc#1170037', 'All nodes not shown by sbd list command', result => 'softfail')
+          if ($num_nodes ne (my $clear_count = () = $sbd_output =~ /clear/g));
```

Comment:
You could remove `$num_nodes` and directly use `if (get_node_number ne ...` here I think.